crowdsec-docs/docs/getting_started/getting_started_on_windows.md

```diff
@@ -35,8 +35,8 @@ We currently support the following Windows services:
  - Windows Firewall: Network scan detection
 
 These directories are created:
- - `C:\Program Files\CrowdSec`: Contains the `crowdsec.exe` and `cscli.exe` executables
- - `C:\Program Files\CrowdSec\config`: Contains all the configuration files
+ - `C:\Program Files\CrowdSec`: Contains the `crowdsec.exe` and `cscli.exe` executables plus the bouncers, collections, parsers and scenarios
```

Comment:
Only the binaries and bouncers are in  `C:\Program Files\CrowdSec`, hub is in `C:\ProgramData`

Reply:
Right, my fault. 
I could fix it after you published this. I need to update and add some other infos.